list_unsigned_rpms

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+from subprocess import check_output, STDOUT, CalledProcessError
+import argparse
+import glob
+
+
+def find_unsigned_packages(target_dir, gpgkey):
+    packages = glob.glob(f"{target_dir}/*.rpm")
+
+    for package in packages:
+        cmd = [
+            'rpm',
+            '-qpi',
+            package
+        ]
+
+        output = check_output(cmd, universal_newlines=True, stderr=STDOUT)
+
+        if gpgkey.lower() not in output:
```

Comment:
Perhaps we should lower case it in `handle_args`?

Reply:
My thinking was to make absolutely sure it's lowered when it's needed to avoid any future changes that might would break this identification of what is signed or not signed.